Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:10.20.1
```

Comment:
Have you considered using a new version of node? Since it doesn't rely on any ahm npm packages or beanstalk?

Reply:
Yeah may as well, will do another PR :)